components/screens/BlogScreen/BlogScreen.js

```diff
@@ -1,12 +1,80 @@
+import { useIsFocused } from '@react-navigation/native';
 import React from 'react';
-import { View, Text } from 'react-native';
+import {useState, useEffect} from 'react';
+import {View, Text, ScrollView, Linking, Button} from 'react-native';
+import { Image } from 'react-native-elements';
+import InAppBrowser from 'react-native-inappbrowser-reborn';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import {Styles} from './BlogStyleSheet';
+
+var blogData = [];
+
+// async function getBlogData() {
+//   let response = await fetch('https://cascade-api.herokuapp.com/blog');
+//   let dataPromise = await response.json();
+//   dataPromise.forEach(data => blogData.push(data));
+
+//   // console.log('OUTSIDE', blogData);
+// }
+// getBlogData();
+
 
 const BlogScreen = () => {
+  
+    const [data, setData] = useState([]);
+    const isFocused = useIsFocused();
+  
+    const getData = async () => {
+      const resp = await fetch("https://cascade-api.herokuapp.com/blog");
+      const data = await resp.json();
+      setData(data);
+      console.log(data);
+    };
+    useEffect(() => {
```

Comment:
## **OPTIONAL:**
ScrollView isn't bad but currently, it doesn't have a loading icon when loading. _(because the component is using `useEffect()`)_. With FlatList it does have a built-in loading icon (see Nataila's PR here).

With no loading icon, it can be confusing for the user to know if the 'calendar screen' is working. If it isn't too difficult, I'd be interested in seeing what a FlatList version of this feature would look like.
If you do this, I'd recommend branching off this current branch and creating a new one called `ab-14-blogScreen-flatlist`, that way if it doesn't work out, you haven't messed anything up here. This is not required but would be interesting to compare the two versions.